packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -0,0 +1,83 @@
+// This script deploys SlashingLibrary, VotingV2 and VotingUpgrader.
+// This script can be run against a mainnet fork by spinning a node in a separate terminal with:
+// HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
+// and then running this script with:
+// yarn hardhat run ./src/upgrade-tests/voting2/0_Deploy.ts --network localhost
+
+const hre = require("hardhat");
+
+import { VotingUpgrader__factory } from "@uma/contracts-frontend/dist/typechain/core/ethers";
+import {
+  Finder,
+  Governor,
+  SlashingLibrary__factory,
+  Voting,
+  VotingToken,
+  VotingV2__factory,
+} from "@uma/contracts-node/typechain/core/ethers";
```

Comment:
nit: similar to the above, we shouldn't need deep imports here, right? Aren't these available at the top level as FinderEthers or SlashingLibraryEthers__factory, etc?

They are aliased, but these types should be available at the package top level (and we should fix if not!).